### PR TITLE
update chai

### PR DESCRIPTION
**Pull request by charlierudolph**

### CODE REVIEW

**Comment by endorama** [comment]
Hello! I've just found this plugin and was asking myself when the support for Chai 2.0.0 will be available!

Any idea on when will this happen? Thanks in advance

**Comment by saadtazi** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -55,6 +55,6 @@
     "underscore": "1.6.0"
   },
   "peerDependencies": {
-    "chai": ">= 1.6.1 < 2"
+    "chai": ">= 2"
```
Comment (saadtazi) [inline package.json]: You might want to use the following instead: `"chai": ">= 1.6.1 < 3"`. Check sinon-chai [package.json](https://github.com/domenic/sinon-chai/blob/master/package.json#l36)

**Comment by charlierudolph** [comment]
@Bartvds any chance this can be merged? Would love to be able to use this with chai 2.0.0

**Comment by kirthar** [comment]
+1
@Bartvds this can be merged?

**Comment by logankoester** [comment]
Yes please!

**Comment by stefanwalther** [comment]
:+1:

**Comment by zckrs** [comment]
@Bartvds can you merge or a problem exist with PR ?

**Comment by nicosommi** [comment]
+1

**Comment by tony** [comment]
Needed this

**Pull request closed without merging**